src/markdown-pages/collect-data/get-started-nerdgraph-api-explorer.mdx

```diff
@@ -10,15 +10,14 @@ tileShorthand:
 redirects:
   - /technology/graphql
 resources:
-  - title: Introduction to New Relic NerdGraph
+  - title: 'Introduction to New Relic NerdGraph'
```

Comment:
Recommend removing New Relic here, so it's just 'Introduction to NerdGraph' - two reasons. One is New Relic is used all over all content, and it becomes a little fatiguing (one of the sweep tasks I just did was to remove extraneous uses of New Relic throughout the dev docs. It's almost always implicit. And this will show up aligned with the article, so it doesn't need the repeat mention of NR.